src/python/pants/bin/pants_loader.py

```diff
@@ -17,6 +17,14 @@ class PantsLoader(object):
   ENTRYPOINT_ENV_VAR = 'PANTS_ENTRYPOINT'
   DEFAULT_ENTRYPOINT = 'pants.bin.pants_exe:main'
 
+  ENCODING_IGNORE_ENV_VAR = 'PANTS_IGNORE_UNRECOGNIZED_ENCODING'
+  BLACKLISTED_ENCODINGS = (
```

Comment:
I'd be in favor of being even stricter, and requiring UTF-8. Do we know that this might cause issues?